omod/src/main/webapp/resources/htmlforms/Covid19-ReferralForm.xml

```diff
@@ -0,0 +1,376 @@
+                                                    <div class="col-md-4">
+                                                        <label>Current</label>
+                                                        <obs conceptId="159395" class="col-md-3" />
+                                                    </div>
+                                                </div>
+                                            </div>
+                                            <div class="col-md-6">
+                                                <label>AVPU(Alert, to Pain, to 
+                                                        Verbal, Unresponsive)</label>
+                                                <div class="row">
+                                                    <div class="col-md-4">
+                                                        <label>Arrival</label>
+                                                        <obs id="arrival" conceptId="159395" answerLabel="arrival" class="col-md-3" />
+                                                    </div>
+                                                    <div class="col-md-4">
+                                                        <label>Current</label>
+                                                        <obs conceptId="159395" class="col-md-3" />
+                                                    </div>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+
+                                <div class="card">
+                                    <div class="card-body">
+                                        <div class="row">
+                                            <div class="col-md-4">
+                                                <label>Investigaion Summary</label>
+                                                <obs conceptId="159395" style="textarea" />
+                                            </div>
+                                            <div class="col-md-4">
+                                                <label>General</label>
+                                                <obs conceptId="159395" style="textarea" />
+                                            </div>
+                                            <div class="col-md-4">
+                                                <label>Key Systemic Exam Finding</label>
+                                                <obs conceptId="159395" style="textarea" />
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <div class="tab-pane active" id="current-management-and-further-plan">
```

Comment:
Remove "active" class among classes. this is causing the tab content  on load to be in the first tab